src/components/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import css from "./Dashboard.module.css";
+import {customHistory} from "../../App";
+import {ErrorMessage} from "../Errors/ErrorMessage/ErrorMessage";
+import {Preloader} from "../Preloader/Preloader";
+
+export class Dashboard extends React.Component{
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            items: [],
+            errorMessage: ''
+        }
+    }
+
+    onLogout = () => {
+        localStorage.removeItem('TOKEN');
+        this.props.onChangeFlag(false);
+        customHistory.push('/signIn')
+    };
+
+    componentDidMount() {
+        this.props.isFetching(true);
+        fetch('/board')
+            .then(response => response.json())
+            .then(response => {
+                this.setState({items: response});
+                this.props.isFetching(false);
```

Comment:
You can move `this.props.isFetching(false)` to `finally` block`

Reply:
plz can be more detailed